### PR TITLE
Ghosted styled to add service block

### PR DESCRIPTION
**Pull request by anthonydillon**

### CODE REVIEW

**Comment by anthonydillon** [comment]
### QA
- Add a service to the canvas
- Begin to create a relation and check the plus-service block fades
- Kill the relation build and see the plus button unfade

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3183/
Test FAILed.

File: jujugui/static/gui/src/app/views/topology/service.js [hunk 1]
```diff
@@ -1531,6 +1531,7 @@ YUI.add('juju-topology-service', function(Y) {
         this.fade({serviceNames: actions.fade});
       }
       if (actions.highlight.length > 0) {
+
```
Comment (hatched) [inline jujugui/static/gui/src/app/views/topology/service.js]: extra space

**Comment by hatched** [comment]
:+1: QA :ok: 
Looks good!

**Comment by makyo** [comment]
Sweet :+1:

**Comment by kadams54** [comment]
LGTM :+1:

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/1087

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3186/
Test PASSed.

**Pull request merged**